### PR TITLE
Disable stop instance

### PR DESCRIPTION
**Pull request by chalfant**
It is sometimes desirable to sysprep a windows machine before creating
an EC2 image. The AWS-approved way to do this is to run
`ec2configservice.exe -sysprep` and let ec2configservice shut down the instance.
This change adds an option to disable the stop instance call issued by packer
so that the user can control when the machine is stopped.

This PR adds a "disable_stop_instance" option to the AWS EBS Builder. 
If it is set to true, packer will not issue a StopInstance API call to stop the build instance. 
The user must manually run the command or remote into the machine and use Ec2ConfigService 
to stop the instance with sysprep.

### CODE REVIEW

File: builder/amazon/ebs/step_stop_instance.go [hunk 1]
```diff
@@ -23,16 +24,20 @@ func (s *stepStopInstance) Run(state multistep.StateBag) multistep.StepAction {
...
-		InstanceIds: []*string{instance.InstanceId},
-	})
-	if err != nil {
-		err := fmt.Errorf("Error stopping instance: %s", err)
-		state.Put("error", err)
-		ui.Error(err.Error())
-		return multistep.ActionHalt
+	var err error
+
+	if s.DisableStopInstance == false {
```
Comment (cbednarski) [inline builder/amazon/ebs/step_stop_instance.go]: Style: this should be written as `if !s.DisableStopInstance`

I think we  should also have an `else` block here that sends a message to the UI indicating that the instance will continue to run and packer will continue to wait until the user intervenes.

File: builder/amazon/common/run_config.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ type RunConfig struct {
 	SourceAmi                string            `mapstructure:"source_ami"`
 	SpotPrice                string            `mapstructure:"spot_price"`
 	SpotPriceAutoProduct     string            `mapstructure:"spot_price_auto_product"`
+	DisableStopInstance      bool              `mapstructure:"disable_stop_instance"`
```
Comment (cbednarski) [inline builder/amazon/common/run_config.go]: Can you add this option to the documentation under `website/` and please include the example use-case that you explained in your PR.

**Comment by cbednarski** [comment]
Thanks for the PR! I left some feedback inline.

**Comment by chalfant** [comment]
I've made the recommended changes. I see that the appveyor CI build is failing but the logs don't seem to indicate anything wrong with my changes. Any suggestions?

**Comment by rickard-von-essen** [comment]
You can ignore AppVeyor :ghost:

**Comment by chalfant** [comment]
Is there anything more that I need to do for this PR?

**Comment by cbednarski** [comment]
LGTM! Thanks @chalfant

**Pull request merged**